src/lib/logger/logger.mli

```diff
@@ -162,6 +163,8 @@ val spam :
   -> ('a, unit, string, unit) format4
   -> 'a
 
+val span : _ log_function
```

Comment:
what is the `span` level exactly?

Reply:
Apologies this commit was some leftover and was reverted